Makefile

```
@@ -100,6 +100,16 @@ dev.sync.requirements: ## Install requirements
 
 dev.sync.up: dev.sync.daemon.start ## Bring up all services with docker-sync enabled
 	docker-compose -f docker-compose.yml -f docker-compose-sync.yml up -d
+	@# Start: Edraak hacks
+	@# TODO: Add this to `base.in` (thus `development.txt`) and rebuild the docker image
+	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install python-bidi==0.4.0'
+	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install wand==0.5.1'
+	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install -e /edx/app/edxapp/edx-platform'
+	@docker exec -it edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && pip install -e /edx/app/edxapp/edx-platform'
+	@make dev.editable-envs
+	@# End: Edraak hacks
+	@# Comment out this next line if you want to save some time and don't care about catalog programs
+	#./programs/provision.sh cache >/dev/null
 
 provision: | dev.provision ## This command will be deprecated in a future release, use dev.provision
```

I think we should re-use for both `dev.up` and `dev.sync.up`:

```suggestion
edraak.dev.up.hacks:
	@# Start: Edraak hacks
	@# TODO: Add this to `base.in` (thus `development.txt`) and rebuild the docker image
	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install python-bidi==0.4.0'
	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install wand==0.5.1'
	@docker exec -it edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && pip install -e /edx/app/edxapp/edx-platform'
	@docker exec -it edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && pip install -e /edx/app/edxapp/edx-platform'
	@make dev.editable-envs
	@# End: Edraak hacks

provision: | dev.provision ## This command will be deprecated in a future release, use dev.provision
```